pkg/sql/colexec/colexecagg/COLEXEC.bzl

```diff
@@ -0,0 +1,70 @@
+# Map between target name and relevant template.
+targets = [
+    ('hash_any_not_null_agg.eg.go', 'any_not_null_agg_tmpl.go'),
```

Comment:
Can you just make this into a helper function? Something like:

    def munge(s):
        if s.startswith('hash_'):
            return s[len('hash_'):].replace('.eg', '_tmpl')
        elif s.startswith('ordered_'):
            return s[len('ordered_'):].replace('.eg', '_tmpl')

    munge('hash_any_not_null_agg.eg.go') => 'any_not_null_agg_tmpl.go'

Maybe something about Bazel is keeping you from doing this -- if so just let me know :)

Reply:
I can give a try, actually I was just following the same logic applied before in older commits but I can try my luck with that change you are suggesting and see what's happening =)

Reply:
Also, do you think is necessary to create a whole new function to only get the `tmpl` target name, for example this one: `('ordered_sum_int_agg.eg.go', 'sum_agg_tmpl.go')` on that one I would need to create an specific `elif` for checking that case?